src/steps.conf.js

```diff
@@ -0,0 +1,36 @@
+const defaultFinishedLoadingConditions = (loaderSelectors) => {
```

Comment:
Maybe we should rename this file to something like `steps.helper.functions.js` (because it looks more like a file with logic then config and we also have a second file with the same name in `test/steps.conf.js` - and it looks like a real config file more)?